featurebyte/session/base_spark.py

```diff
@@ -336,9 +343,13 @@ async def list_procedures(self) -> list[str]:
         return []
 
     async def register_missing_objects(self) -> None:
+        # check storage connection is working
+        session = cast(BaseSparkSession, self.session)
+        session.test_storage_connection()
```

Comment:
After moving this here, storage connection will still be tested on feature store creation, is that right?

Reply:
Yes it will be run every time we re register the udfs